VenueNextSDK-Demo.xcodeproj/project.pbxproj

```diff
@@ -84,15 +99,24 @@
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				AB6E2347228DCA5200EDD290 /* VNLegacy.framework in Frameworks */,
-				AB6E2351228DCA5200EDD290 /* VNPayment.framework in Frameworks */,
-				AB6E234B228DCA5200EDD290 /* VNOrder.framework in Frameworks */,
-				AB6E2349228DCA5200EDD290 /* VNNetworkService.framework in Frameworks */,
-				AB6E2343228DCA5200EDD290 /* VNCore.framework in Frameworks */,
-				AB6E2345228DCA5200EDD290 /* VNCoreUI.framework in Frameworks */,
-				AB5987C822B43C15003E159A /* VNAnalytics.framework in Frameworks */,
-				AB6E234D228DCA5200EDD290 /* VNOrderData.framework in Frameworks */,
-				AB6E234F228DCA5200EDD290 /* VNOrderUI.framework in Frameworks */,
+				6594256922C1741500BCA348 /* VNLegacy.framework in Frameworks */,
+				6594256D22C1741500BCA348 /* VNOrder.framework in Frameworks */,
+				6594255C22C1740B00BCA348 /* VNCoreUI.framework in Frameworks */,
+				6594257122C1741500BCA348 /* VNOrderUI.framework in Frameworks */,
+				6594255E22C1740B00BCA348 /* VNCore.framework in Frameworks */,
+				6594255D22C1740B00BCA348 /* VNNetworkService.framework in Frameworks */,
```

Comment:
Double-check the build phase in the scheme, as this makes it look like the frameworks are being embedded multiple times, or some other kind of duplication is happening.

Reply:
should be good now 
@NFulkerson